app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Any:
+    cashed_functions = {}
+
+    def inner(*args) -> Any:
+        if args not in cashed_functions.keys():
```

Comment:
Not necessary to use .keys here, but it is okay

Reply:
Thank you for your remark.